Makefile

```diff
@@ -68,7 +68,7 @@ all : $(LAYNII)
 .PHONY: all $(HIGH_PRIORITY) $(LOW_PRIORITY) $(LAYNII2)
 
 # =============================================================================
-# LAYNII v2.0.0 programs
+# LAYNII programs
```

Comment:
took the liberty to remove the version number from there: the more instance of the version nnumbe you have in the repo the more annoying it will be to keep them all up to date

Reply:
Good catch, it unnecessary to five version in makefile indeed.